exercises/nth-prime/example.jl

```diff
@@ -0,0 +1,43 @@
+function isPrime(x)
+	i = 2
+	if x == 1
+		return false
+	end
+	if x == 2
+		return true
+	end
+	if x == 3
+		return true
+	end
+	while i <= sqrt(x)
+		if x % i == 0
+		return false
+		end
+		i =i+1
+	end
+	return true
+end
+
+function nthPrime(n)

```

Comment:
I think the name `prime(n)` would be better. It's in line with the canonical data and Julia's conventions.